lab2.md

```diff
@@ -142,10 +142,41 @@ $ docker service create -d \
 stefanprodan/faas-grafana:4.6.3
 ```
 
+> Note: If you're running on Kubernetes, deploy Grafana with the following commands:
+>
+> Run Grafana in OpenFaaS Kubernetes namespace:
+> ```
+> kubectl -n openfaas run \
+> --image=stefanprodan/faas-grafana:4.6.3 \
+> --port=3000 \
+> grafana
+> ```
+>
+> Expose Grafana with a NodePort:
+> ```
+> kubectl -n openfaas expose deployment grafana \
+> --type=NodePort \
+> --name=grafana
+> ```
+>
+> Find Grafana node port address:
+> ```
+> $ GRAFANA_PORT=$(kubectl -n openfaas get svc grafana -o jsonpath="{.spec.ports[0].nodePort}")
+> $ GRAFANA_URL=http://$(minikube ip):$GRAFANA_PORT/dashboard/db/openfaas
```

Comment:
Not everyone is using `minikube` so this would fail for who aren't using it. This is an issue throughout the workshop so we probably need a global place to define the IP.